### ISSUE TITLE
文章页面左边为什么空空的 不展示东西

### ISSUE THREAD

**Issue by qinkangdeid** [dillonzq/LoveIt#184]
如题 还有好大一块地方
![image](https://user-images.githubusercontent.com/17582522/76964103-20b79a80-695d-11ea-8c13-edd20120667f.png)

**Comment by ziobron** [comment]
Yup. I don't know Chinese, but for me also this space on the left is too big. I have edited my _media.scss file to make it more centered. It would be good, however, to fix it when the TOC is displayed only. It should not be moved when TOC is not displayed.

**Comment by dipperkun** [comment]
@dillonzq , 关于这个小问题，有什么开发计划吗？对于宽屏显示器，空白确实很大！

**Comment by dillonzq** [comment]
@dipperkun @qinkangdeid 
有什么具体的建议么?

**Comment by dipperkun** [comment]
个人建议：在正文的右上角做个浮动按钮，负责展开折叠目录。展开时，目录内容位于正文之上，就像你做的那个搜索功能一样。这样的话，正文内容居中显示，占据绝大数屏幕。重点突出，又不浪费空间，毕竟目录不需要一直处于显示状态。

**Comment by dipperkun** [comment]
@dillonzq 以上是我的个人建议！

**Comment by dillonzq** [comment]
@dipperkun  谢谢

**Comment by dipperkun** [comment]
@dillonzq 请问一下，如何控制目录只在文章开头出现，不出现在侧边栏。谢谢！

**Comment by dillonzq** [comment]
@dipperkun 目前是根据视窗宽度自动切换的，还没有设置的方式

**Comment by muyiacc** [comment]
我的建议：在具体的文章页面，左边可以显示一些该文章所在分类下的其他文章，或者相同标签的文章，这两个也可以同时存在。而其他页面还是保存不变，保持居中的样式，显得大气，简洁。
